src/website/src/pages/ec/getting-started/index.mdx

```diff
@@ -92,7 +93,14 @@ There are also other optional css files distributed that can be used, pikaday is
 ECL is also distributing its presets on a CDN, all the files distributed in the package mentioned before are also available there, this is the pattern to reach them:
 `https://cdn{1,2,3 or 4}.fpfis.tech.ec.europa.eu/ecl/{tag}/{system}/{path/to/the/asset}`
 
-Ex: https://cdn1.fpfis.tech.ec.europa.eu/ecl/v3.9.1/ec/styles/optional/ecl-ec-default.css
+Ex:
+
+<a
+  className="ecl-link"
+  href={`https://cdn1.fpfis.tech.ec.europa.eu/ecl/${ECL_VERSION}/ec/styles/optional/ecl-ec-default.css`}
```

Comment:
current link is keeping the "v" before the version, so it should be added there too
![image](https://github.com/user-attachments/assets/1abfc8be-59ad-440b-b5d1-f6aa9618b1bb)